plugins/thunderbolt-power/fu-plugin-thunderbolt-power.c

```diff
@@ -11,7 +11,6 @@
 #include <gio/gio.h>
 #include <glib.h>
 #include <glib/gstdio.h>
-#include <gudev/gudev.h>
```

Comment:
This plugin should be more than just a header change, it should start using the daemon wide client too I think.

Reply:
I didn't want to port this until the bolt stuff was done.

Reply:
Okay.